packages/embark-code-runner/src/index.ts

```diff
@@ -52,101 +50,69 @@ export default class CodeRunner {
     });
   }
 
-  private fireEmbarkJSEvents(code: string) {
```

Comment:
I can understand why we are getting rid of this in favour of a proper solution to the race conditions, however  it probably goes without saying (and just saying to be prudent), we need to be sure that we are removing all other code that relied on these events to fire.

Reply:
Yeah, we found yesterday that those events were needed for the ENS, but we ended up making ENS work again and it doesn't seem that any other module needs those events anymore.